### ISSUE TITLE
Problems on production

### ISSUE THREAD

**Issue by guest73** [Tsmith18256/ng2-inline-href#12]
I pre-compilled my app with the following command:

`ng build --output-path=dist/files/en --aot -prod --env=prod --bh /booq-angular/en/ --i18n-file=src/locale/messages.en.xlf --i18n-format=xlf --locale=en`

Also I deployed it on my Tomcat with /booq-angular path so my final url is http://localhost:7380/booq-angular/en/products.
All the SVG do not work for me even though it works on development environment.
Here is what's in html in production
```
<svg class="table-icon edit-icon" eijsink-color="" viewBox="0 0 24 24">
  <use inlineHref="#edit" xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/products#edit"></use>
 </svg>
```

**Comment by Tsmith18256** [comment]
Hey @guest73, can I get some more clarification on the issue. I see the resulting HTML is showing `/products#edit`, which would usually be correct. Is the issue caused by the prefix on the path? As in, does it work correctly if the href is `/booq-angular/en/products#edit`?

**Comment by guest73** [comment]
> As in, does it work correctly if the href is /booq-angular/en/products#edit?

I tried different things but I do not remember if putting it to /booq-angular/en/products#edit actually helped. Sorry. Probably yes. As the upcoming SVG 2 standard is dropping the requirement for the xlink: prefix I ended up using it just like this, without inlineHref directive.(And I had to separate my icons in a separate SVG file).
```
                <svg id="aside-close-icon">
                    <use href="assets/icons.svg#x-circle"></use>
                </svg>
```
and it worked for me for both development and production, Firefox and Chrome.
